Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-icons bundle size difference from 6.0.2 to bb57506ba271c3e921adc0df4075fba412a2805c
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-icons bundle size difference from 6.0.2 to 5429c8b7f3584cdbdbb93375f7f452cdc39a2afc
No significant bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
